dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -27,7 +27,7 @@
 .script_text
   = text_area_tag('level[dsl_text]', @level.dsl_text || (@level.dsl_default))
 
-- if [ContractMatch, Match, Multi, TextMatch, External].include? @level.class
+- if [ContractMatch, Match, Multi, TextMatch, External, LevelGroup].include? @level.class
```

Comment:
Didn't work on this usage of the editor, but moving the include to within that if looks correct/safe